src/MagnumPlugins/BasisImageConverter/BasisImageConverter.h

```diff
@@ -125,13 +126,31 @@ The @ref PixelFormat::R8Unorm, @relativeref{PixelFormat,R8Srgb},
 @relativeref{PixelFormat,RGBA8Unorm} and @relativeref{PixelFormat,RGBA8Srgb}
 formats are supported.
 
+@subsection Trade-BasisImageConverter-behavior-types Image types
+
+1D images will be saved as 2D images with height 1.
+
+Cube map images can be written but there is currently no way to mark them
+properly in the metadata. Exported files will be 2D array images with faces
+exposed as layers.
+
+3D images will be saved as 2D array images with one layer per depth slice. This
+is necessary because mip levels are always 2-dimensional in Basis compressed
+images. Note that this matches @ref BasisImporter behavior which imports all 3D
+images as 2D array images anyway because of the same Basis limitation.
```

Comment:
I'd simplify this a bit:

```suggestion
The exporter can save 2D and 2D array images. Even though the KTX container
format supports 1D and 3D images, Basis Universal doesn't.

Cube map images can be written but there is currently no way to mark them
properly in the metadata. Exported files will be 2D array images with faces
exposed as layers.
```